hack/quickstart/kubelet.master

```diff
@@ -11,7 +11,8 @@ ExecStartPre=/bin/mkdir -p /etc/kubernetes/cni/net.d
 ExecStartPre=/bin/mkdir -p /etc/kubernetes/checkpoint-secrets
 ExecStartPre=/bin/mkdir -p /etc/kubernetes/inactive-manifests
 ExecStartPre=/bin/mkdir -p /var/lib/cni
-ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/run/kubelet-pod.uuid
+ExecStartPre=/bin/mkdir -p /var/cache/rkt
```

Comment:
@diegs Hmm, I inclined towards putting them in a dir as other packages do. But `/var/cache/rkt` doesn't sound like a best place for it (it's not generated/managed by rkt..). So removing for now.